base_geoengine/geo_model.py

```diff
@@ -98,8 +92,9 @@ def _get_geo_view(self, cursor, uid):
                                       [('model', '=', self._name),
                                        ('type', '=', 'geoengine')])
         if not geo_view_id:
-            raise osv.except_osv(_('No GeoEngine view defined for the model %s') % (self._name,),
-                                 _('Please create a view or modify view mode'))
+            raise osv.except_osv(
+                _('No GeoEngine view defined for the model %s') % self._name,
```

Comment:
If it happens, then the error message won't show, instead there will be an ASCII decode error